### PR TITLE
Make branch warning happen as soon as possible

### PR DESCRIPTION
**Pull request by esoergel**
@benrudolph @dannyroberts
From https://github.com/dimagi/commcare-hq/commit/142e1d5cbba20be98c8e28cba27672a48fc433fa#commitcomment-16832438

### CODE REVIEW

File: fab/fabfile.py [hunk 1]
```diff
@@ -159,8 +156,12 @@ def diff_url(self):
 
     @property
     def deploy_ref(self):
+        print "*"*40, 'ESOE:deploy_ref', "*"*40
```
Comment (dimagimon) [inline fab/fabfile.py]: E226 missing whitespace around arithmetic operator

**Comment by benrudolph** [comment]
thanks, looks good

**Comment by benrudolph** [comment]
no tests

**Comment by dannyroberts** [comment]
Thanks!

**Pull request merged**